neuro_san/internals/run_context/langchain/core/langchain_run_context.py

```diff
@@ -273,7 +271,7 @@ def create_agent(self, prompt_template: ChatPromptTemplate, llm: BaseLanguageMod
         # By skipping this step, our agent functions as a pure LLM-driven system with a defined role,
         # without tool invocation logic influencing its decision-making.
 
-        agent = RunnablePassthrough() | prompt_template | meat | JournalingToolsAgentOutputParser()
+        agent = RunnablePassthrough() | prompt_template | meat
```

Comment:
Don't need the end guy anymore.
We are now processing the output of the chain directly in the factored-out parse_chain_result() message in this class.